src/core/data/teleport_presets.hpp

```diff
@@ -0,0 +1,75 @@
+#pragma once
+#include "gta/enums.hpp"
+typedef struct
+{
+	const char* name;
+	BlipIcons blipsprites;
+} BlipTPlist;
+
+
+
+inline std::vector<BlipTPlist> BlipsBuisness{
+    {"Teleport To Yacht", BlipIcons::Yacht},
+    {"Teleport To Personal Vehicle", BlipIcons::PersonalVehicleCar},
+    {"Teleport To Nightclub", BlipIcons::Nightclub},
+    {"Teleport To Race Finish Test", BlipIcons::RaceFinish},
+    {"Teleport To Facility", BlipIcons::IAAFacility},
+    {"Teleport To Hanger", BlipIcons::AircraftHanger},
+    {"Teleport To Bunker", BlipIcons::Bunker},
+    //	{"Test3", Business},
+    {"Teleport To Terrorbyte", BlipIcons::Terrorbyte},
+};
+inline std::vector<BlipTPlist> BlipsStores{
+    {"Teleport To Ammunation", BlipIcons::AmmuNation},
+    {"Teleport To Barber", BlipIcons::Barber},
+    {"Teleport To Convient Store", BlipIcons::ConvenienceStore},
+    {"Teleport To Tattoo Shop", BlipIcons::TattooParlor},
+    {"Teleport To Stripclub", BlipIcons::StripClub},
+    {"Teleport To LSC", BlipIcons::LosSantosCustoms},
+    {"Teleport To Clothes Store", BlipIcons::Clothes},
+};
+inline std::vector<BlipTPlist> BlipsMC{
+    {"Teleport To Weed Buisness", BlipIcons::Weed},
+    {"Teleport To Cocaine Buisness", BlipIcons::Cocaine},
+    {"Teleport To Meth Buisness", BlipIcons::Meth},
+    {"Teleport To Forged Documents Buisness", BlipIcons::ForgedDocuments},
+    {"Teleport To Counterfeit Money Buisness", BlipIcons::CounterfeitMoney},
+};
+struct teleportlist
+{
+	const char* name;
+	Vector3 Coords;
+	teleportlist(const char* name, float x, float y, float z) :
+	    name(name),
+	    Coords(x, y, z)
+	{
+	}
+};
```

Comment:
Mix and match of field names, some start with a capital letter, none of them have the `m_` prefix.

The struct name is all lower case without an underscore separating the two words, consider changing it as follows:
```cpp
struct teleport_list
{
  const std::string_view m_name;
  Vector3 m_coords;
  
  teleport_list(const std::string_view name, float x, float y, float z) :
    m_name(name),
    m_coords(x, y, z)
  {}
};
```

Reply:
Okay i will, my bad I had a old imgui bbv2 project that i copied the code and tried to optimize i should have work on formattnig for names too